<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEdfaL</class>
 <widget class="QDialog" name="DialogEdfaL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>L波段EDFA控制器</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>-1</y>
     <width>478</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>模式选择：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEdfaLSetPower">
          <property name="currentText">
           <string>15dBm</string>
          </property>
          <item>
           <property name="text">
            <string>15dBm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23dBm</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEdfaLSetMode">
          <item>
           <property name="text">
            <string>ACC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>APC</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>设置最大允许电流</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QSlider" name="horizontalSliderEdfaLSetMaxCurrent">
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="value">
             <number>1600</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxEdfaLSetMaxCurrent">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>160.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="labelEdfaLSetCurrentAndPower">
          <property name="text">
           <string>电流设置</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QSlider" name="horizontalSliderEdfaLSetCurrentAndPower">
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxEdfaLSetCurrentAndPower">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QComboBox" name="comboBoxEdfaLUartList"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonEdfaLUartListRefresh">
            <property name="text">
             <string>扫描串口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonEdfaLUartConnect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEdfaLSendParam">
          <property name="text">
           <string>设置参数</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="labelEdfaLMonitorWorkMode">
          <property name="text">
           <string>工作模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEdfaLMonitorTemperature">
          <property name="text">
           <string>温度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEdfaLMonitorCurrent">
          <property name="text">
           <string>工作电流</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEdfaLMonitorPower">
          <property name="text">
           <string>光功率</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
